packages/components/src/Brand/docs/Brand.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react"
+import { ComponentStory, Story } from "@storybook/react"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
+import { CATEGORIES } from "../../../storybook/constants"
+import { figmaEmbed } from "../../../storybook/helpers/figmaEmbed"
```

Comment:
Drat...just realised these conflict with Storybook 7 :') It's a race!!
Maybe just make sure you don't fix up anything in any stories you copy over so if I copy/paste the fixes later I don't undo anything.